devicetypes/smartthings/smartsense-multi.src/smartsense-multi.groovy

```diff
@@ -120,8 +123,18 @@ private Map parseSingleMessage(description) {
 		handlerName: handlerName,
 		isStateChange: isStateChange,
 		displayed: displayed(description, isStateChange)
-	]
-	log.debug "Parse results for $device: $results"
+	)
+
+	results << createEvent(
+		name: "status",
+		value: value,
+		unit: null,
+		linkText: linkText,
+		descriptionText: descriptionText,
+		handlerName: handlerName,
+		isStateChange: isStateChange,
```

Comment:
I'm specifically talking about `isStateChange`. It's basically assuming that a state change for contact is a state change for status, which I imagine is normally true, but is it always true?